src/services/auth/sponsor/sponsor-router.ts

```
@@ -0,0 +1,118 @@
+import { Router } from "express";
+import { Database } from "../../../database";
+import RoleChecker from "../../../middleware/role-checker";
+import { Role } from "../auth-models";
+import { StatusCodes } from "http-status-codes";
+import { sendEmail } from "../../ses/ses-utils";
+import jsonwebtoken from "jsonwebtoken";
+import { Config } from "../../../config";
+import * as bcrypt from "bcrypt";
+const sponsorRouter = Router();
+
+// Get favorite events for an attendee
+sponsorRouter.get(
+    "/",
+    RoleChecker([Role.Enum.CORPORATE]),
+    async (req, res, next) => {
+        try {
+            const resumeUsers = await Database.REGISTRATION.find(
+                { hasResume: true },
+                { userId: 1 }
+            );
+            if (!resumeUsers) {
+                return res
+                    .status(StatusCodes.NOT_FOUND)
+                    .json({ error: "UserNotFound" });
```

let's also return a different error, maybe smth like "UserResumesNotFound"